lib/bin_conf/Bin_conf.ml

```diff
@@ -615,21 +615,25 @@ type action =
   | Check of input list
   | Print_config of Conf.t
 
+let ( let* ) = Rresult.( >>= )
+
+let ( let+ ) = Rresult.( >>| )
```

Comment:
We don't need `rresult`: 

- Base has these functions
- We depend on ocaml `>= 4.08`, the stdlib has these functions

```suggestion
let ( let* ) = Result.( >>= )

let ( let+ ) = Result.( >>| )
```